### PR TITLE
Updated with latest version information and some minor wording change…

### PR DESCRIPTION
**Pull request by Stealth7777**
…s and links

@kelleyvice-msft Please review and approve

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/page-diagnostics-for-spo.md [hunk 1]
```diff
@@ -26,9 +26,9 @@ description: "Use the Page Diagnostics for SharePoint tool to analyze SharePoint
 This article describes how to use the **Page Diagnostics for SharePoint tool** to analyze SharePoint Online modern and classic site pages against a pre-defined set of performance criteria.  
 
 >[!TIP]
```
Comment (JohanFreelancer9) [inline Enterprise/page-diagnostics-for-spo.md]:
```suggestion
> [!TIP]
```

File: Enterprise/page-diagnostics-for-spo.md [hunk 2]
```diff
@@ -26,9 +26,9 @@ description: "Use the Page Diagnostics for SharePoint tool to analyze SharePoint
...
->**Version 2.0.1 of the tool has been released**. Version **2.0.0** and later includes support for modern pages in addition to classic site pages. If you are unsure which version of the tool you are using, you can select the **About** link or the ellipses (...) to verify your version.
+>**Version 2.0.2 of the tool has been released**. Version **2.0.0** and later includes support for modern pages in addition to classic site pages. If you are unsure which version of the tool you are using, you can select the **About** link or the ellipses (...) to verify your version. Always update to the latest version when using the tool.
```
Comment (JohanFreelancer9) [inline Enterprise/page-diagnostics-for-spo.md]:
```suggestion
> **Version 2.0.2 of the tool has been released**. Version **2.0.0** and later includes support for modern pages in addition to classic site pages. If you are unsure which version of the tool you are using, you can select the **About** link or the ellipses (...) to verify your version. Always update to the latest version when using the tool.
```

File: Enterprise/page-diagnostics-for-spo.md [hunk 3]
```diff
@@ -26,9 +26,9 @@ description: "Use the Page Diagnostics for SharePoint tool to analyze SharePoint
...
 
-The Page Diagnostics for SharePoint tool is a browser extension for Chrome and [Microsoft Edge version 77 and later](https://www.microsoftedgeinsider.com/download?form=MI13E8&OCID=MI13E8) that analyzes both SharePoint Online modern portal and classic publishing site pages. This tool only works for SharePoint Online, and will fail with an error if used on a SharePoint Server site page.
+The Page Diagnostics for SharePoint tool is a browser extension for the new Microsoft Edge (https://www.microsoft.com/en-us/edge) and Chrome browser, that analyzes both SharePoint Online modern portal and classic publishing site pages. This tool only works for SharePoint Online, and cannot be used on a SharePoint system page.
```
Comment (JohanFreelancer9) [inline Enterprise/page-diagnostics-for-spo.md]:
```suggestion
The Page Diagnostics for SharePoint tool is a browser extension for the new Microsoft Edge (https://www.microsoft.com/edge) and Chrome browser that analyzes both SharePoint Online modern portal and classic publishing site pages. This tool only works for SharePoint Online, and cannot be used on a SharePoint system page.
```

File: Enterprise/page-diagnostics-for-spo.md [hunk 4]
```diff
@@ -47,9 +47,9 @@ To provide feedback about the tool, select the ellipsis at the top right corner
 The installation procedure in this section will work for both the Chrome and Microsoft Edge browsers.
 
 > [!IMPORTANT]
-> Microsoft does not read data or page content that is analyzed by the Page Diagnostics for SharePoint tool, and we do not capture any personal information, website or download information. The only information logged by the tool is the tenant name, rule count and whether the support logging option has been enabled when the tool is run. This information is used by Microsoft to understand modern portal and publishing site usage trends and common performance issues to inform product improvements.
+> Microsoft does not read data or page content that is analyzed by the Page Diagnostics for SharePoint tool, and we do not capture any personal information, website or download information. The only identifiable information logged to Microsoft by the tool is the tenant name, counts of the rules that have failed and when the tool is run. This information is used by Microsoft to understand modern portal and publishing site usage trends and common performance issues.
```
Comment (JohanFreelancer9) [inline Enterprise/page-diagnostics-for-spo.md]:
```suggestion
> Microsoft does not read data or page content that is analyzed by the Page Diagnostics for SharePoint tool, and we do not capture any personal information, website, or download information. The only identifiable information logged to Microsoft by the tool is the tenant name, counts of the rules that have failed and when the tool is run. This information is used by Microsoft to understand modern portal and publishing site usage trends and common performance issues.
```

**Comment by kelleyvice-msft** [comment]
I'll incorp the proposed changes from @JohanFreelancer9. Thank you!

**Pull request merged**